### DISCUSSION TITLE
Data sources: Commercial

### DISCUSSION THREAD

**Discussion by melissamonk-NOAA** [pfmc-assessments/CA_quillback_rockfish_2025#4] [General]
List of commercial data sources to include:
1. Catches
- [x] a. PacFIN/CALCOM (PacFIN data provided 7/25/2024. See pfmc-assessments/PacFIN.Utilities#126). For 1984-2023 - landings only
  - [x] Details on remaining decisions can be found in [issue 12](https://github.com/pfmc-assessments/CA_quillback_rockfish_2025/issues/12), [issue 15](https://github.com/pfmc-assessments/CA_quillback_rockfish_2025/issues/15)
- [x] b. Historical reconstruction from EJ based on Ralston Reconstruction. For 1916-1968 - landings only 
Used same from 2021 assessment, which included CA landings from OR/WA waters (which is the same as the assumption Brian used for canary rockfish pfmc-assessments/canary_2023#51)
- [x] c. Historical reconstruction from EJ based on CALCOM. For 1968-1980 - landings only

2. Effort and catch data
- [x] c. ~Commercial trawl logbook data (accessed 11/7/2024).~ Too sparse to include
- [x] d. ~Commercial non-trawl logbook data (provided 11/18/2024).~ Too sparse to include

3. Discards
   
- [x] a. WCGOP (WCGOP data provided by @chantelwetzel-noaa on 5/9/24 with updated observer catch data on 5/15/24).  
  - [x] Need to make decisions on historical discard assumptions. Used Quillback Rockfish (California) and excluded mortality rates from "research".
  - [x] Need to explore commercial WCGOP lengths. Are they the same during recent years when discarding more prominent. 

4. Composition data
- [x] a. PacFin data (provided 7/25/2024). For 1978-2022 (but really 1991-2022)
  - [x] Expand comps based on some of decisions in #6 and #7 
  - [x] b. For age data, PacFIN doesn't have new ages. New pull on Mar 14, 2025 for the most recent (2024) data should get the new data included. 
      - [x] Need to create age comps once new data is pulled and age data is included.

Description of commercial biological data can be found in Data sources: biological

**Comment by melissamonk-NOAA** [comment]
Don Pearson's [tech memo](https://swfsc-publications.fisheries.noaa.gov/publications/TM/SWFSC/NOAA-TM-NMFS-SWFSC-431.pdf) (pg. 73) includes his qualitative description of the reliability of quillback catch estimates in CA. We believe Don has a type-o and meant to say "...although estimates prior to 1992 are generally unreliable."

**Reply by brianlangseth-NOAA** [reply]
They are pretty low before 1992, and so I take this statement to be mostly about 1991 and 1984 based on his figure

**Comment by brianlangseth-NOAA** [comment]
I talked with Vanessa Tuttle about whether quillback occur in ASHOP records and they do not for records south of 42 degrees N. We do not need to deal with ASHOP therefore.

**Comment by brianlangseth-NOAA** [comment]
There are, albeit very limited, commercial bio data in 1978 in pacfin yet catches only go back to 1984. Any sense why this is?

**Comment by melissamonk-NOAA** [comment]
1c.  The trawl logbook data are extremely general and provide no information on quillback rockfish.

**Comment by brianlangseth-NOAA** [comment]
Commit 89b67d3 adds explorations for WCGOP lengths. There are very few samples other than in 2022 (87% of them are for that year) and sample sizes from that year align with what we have for pacfin for that year, though sample size in pacfin is not super high. 

The main point is that the code provides plots showing that lengths in 2022 in WCGOP are comparable to pacfin, and thus it is unlikely we need a separate discard fleet. Choices for what to apply in 2024 will need to be done, but from a standpoint of treatment of discard sizes, our decision to combine appears supported. 

While lengths in other years are of smaller lengths, the limited amount of discards support combining as well.

**Comment by melissamonk-NOAA** [comment]
2d.  The commercial non-trawl logbook data only contains fewer than 10 entries for quillback rockfish. We will not use these data to to inform the assessment. With continued data collection these data could be extremely valuable in the future.

**Comment by brianlangseth-NOAA** [comment]
@melissamonk-NOAA and I discussed commercial ages. Our current pacfin BDS pull only has 2007 ages but Patrick has aged a number of commercial samples. To get samples we need

- [x] check CALCOM to see if Brenda has put in the most recent reads.
- [x] if yes to the above, then email Chantel and ask for newest PacFIN data (request pull on Mar 14)
~- [ ] if not, have Brenda update CALCOM and push to PacFIN~

This will allow us to then generate expanded (and non-expanded) commercial comps

For non-commercial data, brian can generate CAAL and marginal age comps for 
1. All non-commercial fleets
2. CCRFP, and all other non-commercial fleets

**Reply by brianlangseth-NOAA** [reply]
for CCFRP fleet age comps should I try to combine by trips? Looks like sample_id could be used to demarcate fish from something at a finer resolution. Would this field work? 

This would only be used if we have a separate CCFRP fleet, and if use marginals, but it is not hard to do this at this time. For the growth fleet (i.e. CCFRP combined with other things), I will use number of fish as the sample size regardless. Same for CAAL.

**Reply by melissamonk-NOAA** [reply]
All ages are in CALCOM as of 3/12/25.  It might be worth waiting a day for a PacFIN update.

The CCFRP sample_id does not represent a trip.  It represents the year, month and vessel sampled.  It's likely there is more than one trip on a vessel per month.

**Reply by brianlangseth-NOAA** [reply]
Note that in Melissa age databse, the 27 samples for SWFSC boxed are actually commercial samples. Set these to commercail (and thus removed from growth fleet comps) in commit 806b69c

**Comment by JuliaCoates** [comment]
Some years of commercial length comps show poor fits, particularly 2011-2016.  Some explorations we discussed to improve these fits and/or understand their impact include:

- Using different selectivity time blocks
- Removing years with low sample sizes (n<5)
- Increase the weighting of this data source to force the model to fit then implore impacts in other areas of the model.
- Fleets as areas.  However, commercial selectivity was problematic in the FAA structure given the split fleets, time blocks, and lack of data.  The best FAA structure would likely only split the rec fleet and thus this is will not improve commercial length comp fits.  
- Splitting the commercial fleet by disposition (live and dead) rather than area.  Live and dead landings are both very low between 2011-2016 and are almost entirely from the Crescent City port region (see pacfin_landings_port_group_disp_year.png in the data_explore_figs folder).  Splitting the data into these fleets would further reduce a very low sample size.  

Overall, the jagged length comps and poor fits in certain years are the result of low sample sizes and are common for nearshore species.  The best course of action is likely to consider removing the years with very low sample sizes.

**Reply by brianlangseth-NOAA** [reply]
Replying to the last sentence that although this would remove the perceived lack of fit, the model is unlikley to change its estimated trajectory because its not fitting those data points anyway, and so Im more in favor of keeping in and explain that we considered a number of options.
